pkg/logs/auditor/auditor.go

```diff
@@ -116,6 +132,15 @@ func (a *Auditor) Channel() chan *message.Message {
 	return a.inputChan
 }
 
+// SetConfigID set the valid source configuration ID (typically a container ID) for registry
+// update subsequent attempt with a different ID will be ignore for this identifier. It aims
+// to allow only one tailer for the same file to be able to update the registry.
+func (a *Auditor) SetConfigID(identifier, configID string) {
+	if a.identifiersChan != nil {
+		a.identifiersChan <- &Identifiers{identifier, configID}
```

Comment:
Currently it won't happen because `Auditor.SetConfigID` potential callers are -as of today- all shut down before the auditor itself (https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/agent.go#L93-L139). Message are also submitted to the auditor without further protection because of this ([here](https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/sender/batch_strategy.go#L103) & [here](https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/sender/stream_strategy.go#L33)).
As far as I know there is no reason for this to change in the future.

Reply:
got it, thanks